rollup.config.base.js

```diff
@@ -1,3 +1,5 @@
+
+
```

Comment:
```suggestion
```

remove unnecessary new lines

Reply:
(also yes, I think we should add `prettier` for formatting, which will be easier now that we no longer have to integrate with deprecated `tslint`)

Reply:
I just tried prettier and it insists on putting opening braces on the same line. >_<

We are long past the time of printing code on transparent slides to project them on a wall trough a contraption of lights and mirrors, so there is no reason for that code style. :)

Reply:
tbh, I haven't seen your style of opening braces on a new line in like over a decade -- it's very rare 😅 

the main thing with `prettier` and other tools like `standard` (also `isort` et al) is that there is a _consistent_ style, so all code looks the same automatically, with no manual checks needed. (see also [`prettier`'s Philosophy](https://prettier.io/docs/en/option-philosophy), which is very similar to [`standard`'s](https://standardjs.com/#why-should-i-use-javascript-standard-style), albeit sometimes favoring different styles)

braces specifically there is no config option for